src/stream/repeat.rs

```diff
@@ -0,0 +1,48 @@
+use core::marker;
+
+
+use stream::Stream;
+
+use {Async, Poll};
+
+
+/// Stream that produces the same element repeatedly.
+#[must_use = "streams do nothing unless polled"]
+pub struct RepeatStream<T, E>
```

Comment:
Could this be called `Repeat` instead? I see that `IterStream` doesn't follow this convention but I think that may be a mistake.